### PR TITLE
Add server application

### PR DESCRIPTION
**Pull request by kpustelnik**
This pull requests adds a sample server application for the multiplayer functionality mentioned in #6 .

I've decided to use sockets - please let me know if you think that's a bad idea.

After we add build CI/CD we should be able to just run that on any OS using [Docker](https://docker.com).

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM python:3.9
```
Comment (jaanonim) [inline server/Dockerfile:1]: For consistancy let's use the same version fo python on server and client, that is 3.10.
And for better image size use alpine.
[3.10-alpine3.18](https://hub.docker.com/layers/library/python/3.10-alpine3.18/images/sha256-4b062a3861ac865436439499d9a03e055871a0f465833987d23b67364a35a38f?context=explore)

File: server/src/app.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+		except Exception as e:
+			print(e)
+			break
+		
+	c.close()
+
+# Stworzenie socketa
+s = socket.socket()
+port = 12345
```
Comment (jaanonim) [inline server/src/app.py:37]: Maybe use env variables for port setting.

File: server/src/app.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+s.bind(('0.0.0.0', port))		
+print ("Socket bound to port %s" %(port))
+
+s.listen()	
+
+while True:
+	c, addr = s.accept()	
+	print ('Incoming connection from', addr)
+	t1 = threading.Thread(target = handle_client, args=(c, ))
+	t1.start()
```
Comment (jaanonim) [inline server/src/app.py:47]: Missing newline on end of file

**Review by jaanonim — CHANGES_REQUESTED** [review]

File: server/src/app.py (hunk 3 above)
Comment (kpustelnik) [reply server/src/app.py:47]: Do you mean `<br />`?

File: server/src/app.py (hunk 2 above)
Comment (kpustelnik) [reply server/src/app.py:37]: How do I use env in python?

File: server/Dockerfile (hunk 1 above)
Comment (kpustelnik) [reply server/Dockerfile:1]: Fact. Hope that's compatible!

File: server/src/app.py (hunk 3 above)
Comment (jaanonim) [reply server/src/app.py:47]: No just `\n`

File: server/src/app.py (hunk 2 above)
Comment (jaanonim) [reply server/src/app.py:37]: you can use `os.environ['PORT']`

Comment (kpustelnik) [reply server/src/app.py:37]: Thanks! On it.

**Review by jaanonim — APPROVED** [review]
LGTM

**Pull request merged**